app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -24,6 +30,7 @@ class ApiController < ActionController::Base
         skip_before_filter :verify_authenticity_token, unless: :session_user?
         skip_before_filter :verify_authenticity_token, only: :options
         before_filter :set_cors_preflight_headers, only: :options
+        before_filter :set_cors_headers
         after_filter :set_cors_headers
 
```

Comment:
Could we remove the `after_filter` for cors and date_headers here, or is it still needed ?